components/brave_rewards/browser/rewards_service.cc

```diff
@@ -48,6 +48,8 @@ void RewardsService::RegisterProfilePrefs(PrefRegistrySimple* registry) {
                                   base::TimeDelta::FromDays(7));
   registry->RegisterTimeDeltaPref(prefs::kRewardsBackupNotificationInterval,
                                   base::TimeDelta::FromDays(7));
+  registry->RegisterTimeDeltaPref(prefs::kRewardsNotificationStartupDelay,
```

Comment:
I think we should increase this, so that slow computers have enough time as well. Maybe 30?

Reply:
I agree, I don't think this needs to run so close to when things may still possibly be initializing. Done